.github/workflows/main.yml

```diff
@@ -95,8 +95,12 @@ jobs:
           python -m pip install tox
           tox --version
 
-      - name: Run tox
-        run: tox
+      - name: Run unit tests
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 5
```

Comment:
Do we need the time out here?

Reply:
Looks like we're setting it to 10 minutes on line 72, so I guess I'll change it to be consistent.

Reply:
Sounds like we write the exact number in different places, maybe we should consider consolidating it into one place, like a variable or something? Not a hard restriction